packages/app/src/components/Menu.tsx

```diff
@@ -1,11 +1,19 @@
 import { useMenu, useMenuItem } from "@react-aria/menu";
-import { Item } from "@react-stately/collections";
+import { mergeProps } from "@react-aria/utils";
+import { Item as RootItem } from "@react-stately/collections";
 import type { TreeProps, TreeState } from "@react-stately/tree";
 import { useTreeState } from "@react-stately/tree";
 import type { ItemProps, Node } from "@react-types/shared";
+import cx from "classnames";
 import type { FC, Key } from "react";
 import { useEffect, useRef } from "react";
 
+const style = {
+  menu: `rounded-md shadow-sm min-w-[120px] text-gray-300`,
+  menuItem: `flex items-center gap-2 px-4 py-2 focus-ring cursor-pointer text-sm first:rounded-tl-md first:rounded-tr-md
+  last:rounded-bl-md last:rounded-br-md`,
+};
+
 type ListItemProps = {
   item: Node<any>;
```

Comment:
I know it's not related to your change, but its possible to add type here?

Reply:
✅